<?xml version="1.0"?>
<launch>
  <arg name="calibration" default="" />
  <node pkg="but_velodyne_proc" type="ground_map" name="but_velodyne_proc_groundmap_node" output="screen">
    <remap from="points_in" to="/velodyne_points" />
    <remap from="map2d_out" to="/velodyne/map2d" />
    <param name="max_road_irregularity" value="0.04" />
    <param name="max_height_diff" value="0.1" />
    <param name="noise_filter" value="1" />
    <param name="ground_prob" value="0.75" />
    <param name="obstacle_prob" value="0.4" />
    <!--param name="frame_id" value="/velodyne" /-->
    <param name="frame_id" value="/odom" />
  </node>
</launch>
